docs/nydus-failover-upgrade.md

```diff
@@ -0,0 +1,68 @@
+The following document has been translated from Chinese to English:
+
+Currently, nydusd supports failover and hot upgrade under fusedev and fscache modes, specifically:
+
+- After a nydusd process crashes, a newly started nydusd process can automatically recover the mount points and other filesystem state information of the crashed nydusd without affecting the normal operation of containers.
+- Users can upgrade the version of nydusd without affecting the normal operation of containers. The new nydusd process of the upgraded version can automatically recover the mount points and other filesystem state information of the old nydusd process.
```

Comment:
Ditto, `after the old nydusd exits and the new nydusd starts, it can still serve container I/O operations`.